### DISCUSSION TITLE
vscode copilot unstable, sometime doesn't work

### DISCUSSION THREAD

**Discussion by weiwongfaye** [community/community#36152] [Copilot Conversations] [answered]
today, I found the vscode + copilot doesn't work on my mac. already installed the latest vscode (Version: 1.72.2), latest copilot(v1.53.7011) and signin correctly.
saw below log from copilot output
```
[INFO] [ghostText] [2022-10-14T15:04:38.972Z] Cancelled after extractPrompt
[INFO] [ghostText] [2022-10-14T15:04:38.972Z] Cancelled after extractPrompt
[INFO] [ghostText] [2022-10-14T15:04:38.972Z] Cancelled after extractPrompt
[INFO] [ghostText] [2022-10-14T15:04:38.972Z] Cancelled after extractPrompt
[INFO] [ghostText] [2022-10-14T15:04:38.973Z] Cancelled after extractPrompt
[INFO] [ghostText] [2022-10-14T15:04:38.974Z] Cancelled after extractPrompt
[INFO] [ghostText] [2022-10-14T15:04:38.974Z] Cancelled after extractPrompt
[INFO] [ghostText] [2022-10-14T15:05:55.132Z] Cancelled after requestMultiline

```

**Comment by FahdSeddik** [comment]
Hey @weiwongfaye 
I think you might find an answer [here](https://github.com/community/community/discussions/6785). Hope this helps!

**Reply by weiwongfaye** [reply]
thanks @FahdSeddik, but I do check that thread and it doesn't work for me. 
yesterday, I got it worked for a while but today it doesn't work again.  (nothing changed of vscode and the copilot plugin).  my networks is also fine, it seems to me there are some issue with connecting to the copilot server recently
<img width="1144" alt="vscode_copilot" src="https://user-images.githubusercontent.com/1755044/196012669-5c42db92-8b4d-4f72-bb32-3e4f6368b496.png">

**Comment by weiwongfaye** [comment, marked as answer]
aha, I think I found the cause.  I was login to vscode+copilot with same account on both my personal mac and my work Mac. it seems  that my company's network settings blocked the connection to copilot and that affect the behavior of my personal mac too. after I logout the one from my work Mac, it starts working on my personal one now.
